repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEntityStoreV2.java

```diff
@@ -631,6 +641,10 @@ public EntityMutationResponse deleteByIds(final List<String> guids) throws Atlas
 
             AtlasAuthorizationUtils.verifyDeleteEntityAccess(typeRegistry, entityHeader, "delete entity: guid=" + guid);
 
+            if (AtlasConfiguration.NOTIFICATION_ATLAS_DISTRIBUTED_TASKS_TOPIC_NAME.getBoolean()) {
```

Comment:
AtlasConfiguration.NOTIFICATION_ATLAS_DISTRIBUTED_TASKS_TOPIC_NAME.getBoolean() 

I guess this should be
AtlasConfiguration.ATLAS_DISTRIBUTED_TASK_ENABLED.getBoolean()